### PR TITLE
release-notes.txt: added release notes for v4.4.0

### PR DESCRIPTION
**Pull request by hilmarm**
Signed-off-by: Hilmar Magnusson <hilmar.magnusson@bisdn.de>

### CODE REVIEW

**Review by KanjiMonster — CHANGES_REQUESTED** [review]

File: release-notes.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+! New in this release
+- Edgecore AS4630-54PE (48 x 1GbE RJ45 PoE, 4 x 25GbE SFP28 and 2 x 100GbE QSFP28)
+  added as supported platform.
+- Updated Linux kernel to 5.15
+- Updated FRR to version 8.1
+- Experimental feature: Support for spanning tree protocol (STP) on bonds.
+
+! Bug fixes
+- basebox: Fixed disabling VLANs on ports currently subscribed to multicast groups
+- basebox: Fixed local cache flush on vlan removal
```
[the remark this answers is not in the document]
Comment (hilmarm) [reply release-notes.txt]: fixed together with the other commit, squashed the fix commits into one

File: release-notes.txt [hunk 2]
```diff
@@ -0,0 +1,72 @@
+BISDN Linux
+4.4.0 Release Notes
+Date: 2021-12-15
+
+! New in this release
+- Edgecore EPS202 / AS4630-54PE (48 x 1GbE RJ45 PoE, 4 x 25GbE SFP28 and 2 x 100GbE QSFP28)
+  added as supported platform.
+- Updated Linux kernel to 5.15.
+- Updated FRR to version 8.1.
+- Experimental feature: Support for spanning tree protocol (STP) on bonds.
```
Comment (jklare) [inline release-notes.txt:10]:
```suggestion
- Experimental feature: Support for spanning tree protocol (STP) on bonds.
- Add git package to all images
```

Comment (hilmarm) [reply release-notes.txt:10]: added!

File: release-notes.txt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Add git package to all images
```
Comment (KanjiMonster) [inline release-notes.txt:11]:
```suggestion
- Added git package to all images.
```

**Comment by jklare** [comment]
please do not forget to squash the commits

**Comment by hilmarm** [comment]
> please do not forget to squash the commits

Squashed everything into one commit

**Review by KanjiMonster — APPROVED** [review]
lgtm now

**Pull request merged**